sdk/identity/azure-identity/setup.py

```diff
@@ -73,7 +73,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.0.0",
         "cryptography>=2.1.4",
-        "msal<2.0.0,>=1.3.0",
+        "msal<1.5.0,>=1.3.0",
```

Comment:
Is this a breaking change?

Reply:
No. The latest `msal` is 1.4.3. The purpose of this is to ensure the workaround for `msal` not accepting encrypted private keys isn't broken by the next minor version, which will accept encrypted keys. That's expected to land before we want to mark 1.5.0 stable, so I'll revert this change before then.